code/modules/spells/spell_types/touch/smite.dm

```diff
@@ -57,3 +57,15 @@
 	icon = 'icons/obj/weapons/hand.dmi'
 	icon_state = "disintegrate"
 	inhand_icon_state = "disintegrate"
+
+/obj/item/melee/touch_attack/smite/suicide_act(mob/living/user)
+	// Mostly copy-pasted from /obj/item/grenade/c4/suicide_act(mob/living/user) because I doubt we'd need "thing explodes on suicide_act" to be generalized
```

Comment:
this doesnt really seem useful as a comment

Reply:
Fair enough I'll get rid of it when I get home

Reply:
I forgot to do this because I got injured on my way home that day, I'll remember tomorrow